Clicker/Views/OptionsView.swift

```diff
@@ -15,6 +15,9 @@ protocol SliderBarDelegate {
 
 class OptionsView: UIView, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
     
+    //MARK: Constants
+    let questionOptionCellId = "questionOptionCellId"
```

Comment:
We should probably be consistent on where we're placing constants. 
I think one way we could do it is place all the cell identifiers into the Constants file under the ReuseIdentifiers struct but keep constants like CGFloats as instance variables of the class since they're a bit more specific to the class.